integration_tests/test_coverage.py

```diff
@@ -80,7 +80,7 @@ def _get_current_coverage(coverage_config, no_cleanup, test_scope):
 
     additional_exclude_path = coverage_config["exclude_path"]
     if additional_exclude_path:
-        llvm_cov_command += f" --ignore-filename-regex {additional_exclude_path}"
+        llvm_cov_command += f" --ignore-filename-regex \"{additional_exclude_path}\""
```

Comment:
And all of that just because the code uses `shell=True` instead of just arranging the parameters in an array :/